website/src/content/api-reference/commercetools-frontend-application-shell.mdx

```diff
@@ -73,6 +73,35 @@ export default EntryPoint;
 | `render`                         | `func`             |            (*)           | -       | The render function is called when the `<ApplicationShell>` is ready to render the actual application. This is the case when the required data (user, project) has been fetched and the application context has been initialized. It's **recommended to use the `children`** prop to benefit from a simpler setup.                                                                                                                                                                                                                                  |
 | `apolloClient`         | `ApolloClient`           |                       | -       | An optional instance of [ApolloClient](https://www.apollographql.com/docs/react/) to be used instead of the default one. This is usually the case when you need to configure the Apollo cache. See [`createApolloClient`](#createapolloclient).                                                                                                                                                                                                                                        |
 
+
+## ApplicationPageTitle
+
+Provides Custom Applications the ability to display a more useful (Human readable) page title in the browser tab based on the page location. The following format is used:
+
+```
+<Entry-point-uri-path> - <project-key> - Merchant Center
+```
+
+You can optionally add more contextual information to the default page title by rendering the `<ApplicationPageTitle>` component and passing additional values.
+The values are prepended to the default page title and concatenated with a `-` separator. If each additional value exceeds 24 characters length, it will be truncated in the middle.
+
+The application shell now has an instance of the `<ApplicationPageTitle>` component. Page titles in the browser tabs will now automatically get more readable titles.
```

Comment:
Neat 👏

I feel this part is already covered in the text above. Also, I feel this wording applies better to release notes than docs. 
```suggestion
The application shell has an instance of the `<ApplicationPageTitle>` component.
```